white-paper/wp.tex

```diff
@@ -222,31 +222,38 @@ \section{Architecture}
 System contents the following modules:
 \begin{description}
     \item[Graph Generator.]
-    Based on predefined settings, module should create bi-directional graph. Nodes are the physical servers which contain one or more repositories. Node state must be set based on settings with some minor random delta:
+    Based on predefined settings, module should create directed graph. \textbf{Nodes} are the physical servers which contain one or more repositories. Node state must be set up based on settings with some minor random delta:
     \begin{itemize}
         \item Read operation speed (MB/sec)
         \item Write operation speed (Mb/sec)
-        \item State (Active/Down)
+        \item State (Active/Down/Outdated)
     \end{itemize}
-    Edges of graph are routes from one Node to another. They must have their own configuration properties:
+    \textbf{Edges} of graph represent network connections from one Node to another. They must have their own configuration properties:
     \begin{itemize}
-        \item Throughput (max Mb/sec)
+        \item Throughput (max Mb/sec)\todo Or it can be on vertex level only.
         \item State (Active/Down)
     \end{itemize}
+    If edge is not active, system must find another way to get the target node.\par
+    In corner case there may be nodes without GitHub repositories, serve as \textbf{routers}. Such routers can have different strategies and have not storage (can't serve as a final destination of the request.\par
```

Comment:
@IlyaMoskva I don't think we need to mention repositories or Github here - a server may run any arbitrary logic code, not related to git ropos

Reply:
Yes, @g4s8 , good point, I've changed the text.